### ISSUE TITLE
Blackplanet false positive

### ISSUE THREAD

**Issue by raphuchur** [sherlock-project/sherlock#113]
Blackplanet is giving false positives. (request from germany)
@Czechball you added this in #81 ; maybe you are able to fix it?

**Comment by Czechball** [comment]
Hello, I also noticed these false positives. Blackplanet has a very weird way of dealing with non-existent accounts... It basically shows you a broken user edit gui. So I tried to bodge the fail detection by adding "edit profile" text in errorMsg, but it's probably not working... I'm not really an expert in this, so idk why it isn't working.

**Comment by filipceglik** [comment]
Hi, I've looked on this issue and the only way I found to not get false positives is the angular errors. When you check for a non-existent user, angular responds with "TypeError: Cannot read property 'username' of undefined". I've checked this for multiple users and this error only occurs with non-existent usernames. Unfortunately I don't know how to extract this data in python, although I would be more than happy if someone gave me a tip and I will handle this issue. 

![image](https://user-images.githubusercontent.com/17405814/51073919-ecb95900-1677-11e9-82d3-d592bcde6ddc.png)

**Comment by masta2898** [comment]
@filipceglik, this is the part of the Angularjs code that is ran by your browser. Unfortunately, the Python query library cannot run JS code.

**Comment by hoadlck** [comment]
I took a look at this, and I am not seeing how to detect a non-existent profile.  I will try to look at this tomorrow.

Should we continue to have a site included if detection does not work?

**Comment by hoadlck** [comment]
I looked into this in more detail, and...oh boy.  

I captured the response text that Sherlock receives for http://blackplanet.com/blue (claimed) and http://blackplanet.com/bluedfdf (unclaimed).  They are exactly the same!

The site must be rendering all of the different content using Javascript in the browser.  So, I don't see a good way to distinguish between the results.  I have used Selenium to do browser automation. That would allow us to get all of the final text.  I have not run ChromeDriver headless, but it does seem to be supported.  But, it would require that the user have Chrome installed.

I think that we need to disable this site.  Sherlock cannot detect an unclaimed username, so it will just be confusing.  Thoughts?
